tests/unit/CMakeLists.txt

```diff
@@ -15,20 +15,41 @@ endif()
 
 ## base tests
 file(GLOB tests_high_five_src "*base.cpp")
+file(GLOB tests_high_five_src_easy "*easy.cpp")
 
-add_executable(tests_high_five_cpp11_bin ${tests_high_five_src})
+add_executable(tests_high_five_cpp11_bin ${tests_high_five_src} ${tests_high_five_src_easy})
 target_link_libraries(tests_high_five_cpp11_bin
 ${Boost_UNIT_TEST_FRAMEWORK_LIBRARIES} ${HDF5_C_LIBRARIES})
 
 if(HIGHFIVE_PARALLEL_HDF5)
-    target_link_libraries(tests_high_five_cpp11_bin ${MPI_C_LIBRARIES})
+  target_link_libraries(tests_high_five_cpp11_bin ${MPI_C_LIBRARIES})
 endif()
 
 target_compile_options(tests_high_five_cpp11_bin PRIVATE "${HIGHFIVE_CPP_STD_FLAG}")
 
 add_test(NAME tests_high_five_cpp11 COMMAND tests_high_five_cpp11_bin)
 
+if(USE_EIGEN)
+  list(APPEND CMAKE_MODULE_PATH "/usr/share/cmake-2.8/Modules/")
```

Comment:
Why is this `list(APPEND ...)` required? This path is already in the searched paths. Please remove it.

Reply:
The only reason that this is included is that, without this, the default build environment is failing to find Eigen.

Reply:
I agree this is a hack that we should try to remove. I can try to look it up

Reply:
Could you please detail in which conditions you need this `APPEND CMAKE_MODULE_PATH`? at least the following information, Linux distribution, cmake version, eigen installation and version, cmake command to build highfive

Reply:
For the HighFive's Travis/Linux environment for which no compiler is specified (i.e not `GCC_VERSION=7` or `CLANG_VERSION=...`).

I cannot reproduce a failure to find Eigen on any of my systems.

Reply:
That may be solvable with an environmental variable configured in the `.travis.yml` file rather than hardcoding it here.

Reply:
I'm all for it! I could not find how to, do you know?